typecats/exceptiongroups/pytest_utils.py

```diff
@@ -0,0 +1,133 @@
+"""WARNING: This module is experimental.
```

Comment:
you should probably put this under `tests`. At least historically, we've avoided putting non-runtime dependencies within the main source directory. You run the risk of people getting `ImportErrors` if they do anything you didn't expect.

Reply:
Yeah, this one's a bit tricky. I intend to use `raises_in_group` in other projects' test suites which is why I included it here (also due to lack of interest in maintaining a separate package for only a few functions). My idea is that including `pytest` in the module name would give people a pretty good idea of what to expect if they were to import it. Do you think this logic holds up? Since I'll be moving the exceptiongroups utils module into the root typecats package, I would have to do the same with this one. Perhaps prefixing it with an underscore-- `typecats/_pytest_utils.py`?

If you feel like this doesn't belong here, I won't take offense! Perhaps something like the_truck is a better place for it?

Reply:
After some consideration I have decided that I don't want to maintain this as a public API. I'm sure that the Python community will come up with better utilities over time and I don't want to have to deprecate this in a public package whenever that happens.

Reply:
i just want to validate your comprehensive thought process here.  I think in XOi's case, maybe `the_truck` is the right place to put this, but either way, I can see now where you were going with this. 

And yes, i agree that if you had decided to keep it here, putting it under a `_` prefix would be a good plan to try to keep it out of everybody's way.